code/modules/projectiles/ammunition/caseless/rocket.dm

```diff
@@ -1,16 +1,17 @@
+/obj/item/ammo_casing/caseless/rocket
+	name = "pm-9HE"
+	desc = "An 84mm High Explosive rocket. Fire at people and pray."
+	caliber = "84mm"
+	icon_state = "srm-8"
+	projectile_type = /obj/item/projectile/bullet/a84mm_he
 
-/obj/item/ammo_casing/caseless/a84mm
-	desc = "A 84mm HE rocket."
+/obj/item/ammo_casing/caseless/rocket/hedp
+	name = "pm-9HEDP"
+	desc = "An 84mm High Explosive Dual Purpose rocket. Pointy end toward mechs."
 	caliber = "84mm"
 	icon_state = "84mm-hedp"
 	projectile_type = /obj/item/projectile/bullet/a84mm
 
-/obj/item/ammo_casing/caseless/srm8
-	desc = "A 84mm HE rocket."
-	caliber = "84mm"
-	icon_state = "srm-8"

```

Comment:
shame on you for removing mech warrior refrence

Reply:
It wasn't visible in game anyways.